src/query/xasl.h

```diff
@@ -1034,6 +1034,7 @@ struct xasl_node
   OID *class_oid_list;		/* list of class/serial OIDs referenced in the XASL */
   int *class_locks;		/* list of locks for class_oid_list. */
   int *tcard_list;		/* list of #pages of the class OIDs */
+  int includes_tde_class;	/* whether there is a tde class in class_oid_list */
```

Comment:
If possible I recommend using xasl.flag.

Reply:
It could be a better way and seems to be possible. I'll consider it and let you know.